src/actions/createInitialMessage.ts

```diff
@@ -26,7 +28,7 @@ export const createInitialMessage = async (): Promise<string | void> => {
     }
 
     let baseMessage = `*${pull_request.user?.login}* is requesting your review on <${pull_request._links.html.href}|*${pull_request.title}*>`;
-    if (!!pull_request.body) {
+    if (!!pull_request.body && verbose) {
       baseMessage = `${baseMessage}\n>${pull_request.body}`;
```

Comment:
So only if there is a PR body and verbose is on (default is true) then it'll add the PR body into the slack mesage.
Setting `verbose: false` will have the behavior I imagine of just posting the title message with a link.